app/models/package_manager/maven_manager/spring_libs.rb

```diff
@@ -0,0 +1,61 @@
+module MavenManager
```

Comment:
I'm not sure about this module setup. I would like it to live under the PackageManager module but was having difficulties with the Base package manager class referencing things outside of its module.

Reply:
I think rails would want it to be
```
module PackageManager
  module MavenManager
    class SpringLibs < Maven
``` 
?
Or maybe even 
`class PackageManager::MavenManager::SrpingLibs` ?